pkg/probe/probes.go

```diff
@@ -72,6 +73,23 @@ func ping(target string, timeout time.Duration) (string, error) {
 	})
 }
 
+func pingWithTCP(target string, timeout time.Duration) error {
+	tcpingProbe := tcping.NewProbe(tcping.GetInterface(""), 3000*time.Millisecond, true)
+
+	// Port 80 does not need to be open, this just check the tcp handshake with
```

Comment:
Really? What if firewall straight up rejects our request, then we don't even get to anything listening on TCP, do we?

Reply:
Well maybe reaching the firwall is good enough :-)

Reply:
The target may always drop the request with no response right? I think no solution is 100%. Using TCP port 80 should have good chance, as it's usually allowed for HTTP traffic...